components/table/ApplicationTable.tsx

```diff
@@ -140,19 +149,36 @@ const ApplicationTable: FC<ApplicationTableProps> = ({
         header: 'Reviewer',
         id: SEARCH_PARAM_REVIEWER
       }),
+      columnHelper.accessor('internalReview.reviewerPercentile', {
+        cell: (info) => info.getValue() || 'N/A',
+        header: 'R %',
+        id: 'percentile',
+        filterFn: (row) => {
+          const value = row.original.internalReview?.reviewerPercentile
+          return (
+            value !== undefined &&
+            value !== null &&
+            parseInt(value.toString(), 10) <= parseInt(percentileThreshold, 10)
+          )
+        }
+      }),
       columnHelper.display({
         id: 'forms',
         header: 'Forms',
         cell: (info) => (
           <Flex gap="1">
             <AdminScoringDialog data={info.row.original} user={{ email: email, role: role }} />
             <ReviewerScoringDialog data={info.row.original} userEmail={email} />
-            <UgTutorDialog data={info.row.original} user={{ email: email, role: role }} />
+            <UgTutorDialog
+              data={info.row.original}
```

Comment:
Could it be worth extracting `info.row.original` in a variable called `data` to improve readability?